stylesheet.css

```diff
@@ -0,0 +1,310 @@
+* {
+  /* border: 1px black dotted; */
+  /* overflow: hidden; */
+}
+
+body {
```

Comment:
What made you decide to put the globals in the body and not in "*". I struggle with this decision myself.

Reply:
I actually first had it target * {} but then went with body because I felt like if I apply it to everything I may forget that it affects everything (wow much logic) and then don't really understand why e.g. a class that I haven't specifically styled would be formatted in a way I don't expect.
I think if you have a small css it's totally valid to use * – or with something that is really relevant to EVERYTHING, such as font-families. I usually do that to get rid of the serifs EVERYWHERE